gaudi/README.md

```diff
@@ -0,0 +1,103 @@
+**Note:** This step will reboot the nodes, it is recommended to do this in the first step.
+
+The default kernel firmware search path `/lib/firmware` in RHCOS is not writable. Command below can be used to add path `/var/lib/fimware` into the firmware search path list.
+```
+oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/gaudi/gaudi_firmware_path.yaml
+```
+
+## Label Gaudi Accelerator Nodes With NFD
+NFD operator can be used to configure NFD to automatically detect the Gaudi accelerators and label the nodes for the flowing provisioning steps.
+```
+oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/gaudi/gaudi_nfd_instance_openshift.yaml
+```
+Verify NFD has labelled the node correctly:
+```
+oc get no -o json | jq '.items[].metadata.labels' | grep pci-1da3
+
+"feature.node.kubernetes.io/pci-1da3.present": "true",
+```
+NFD detects underlying Gaudi Accelerator using its PCI device class and the vendor.
+
+## Install HabanaAI Operator on Red Hat OpenShift
+### Installation via web console
+Follow the steps below to install HabanaAI Operator using OpenShift web console:
+1.	In the OpenShift web console, navigate to **Operator** -> **OperatorHub**.
+2.	Search for **HabanaAI Operator** in all items field -> Click **Install**.
+### Verify Installation via web console
+1.	Go to **Operator** -> **Installed Operators**.
+2.	Verify that the status of the operator is **Succeeded**.
+
+### Installation via Command Line Interface (CLI)
+```
+oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/gaudi/gaudi_install_operator.yaml
+```
+
+### Verify Installation via CLI
+Verify that the operator controller manager pod is up and running:
+```
+oc get pods -n habana-ai-operator
+
+NAME                                  READY   STATUS    RESTARTS   AGE
+controller-manager-6c8459d9cb-fqs8h   2/2     Running   0          25m
+```
+
+## Creating Habana AI Operator DeviceConfig Instance
+To create a Habana Gaudi device plugin CR, follow the steps below.
+
+### Create CR via web console
+1.	Go to **Operator** -> **Installed Operators**.
+2.	Open **HabanaAI Operator**.
+3.	Navigate to tab **Device Config**.
+4.	Click **Create DeviceConfig** -> set correct parameters -> Click **Create**. To set correct parameters please refer [Using RedHat OpenShift Console](https://docs.habana.ai/en/latest/Orchestration/HabanaAI_Operator/Deploying_HabanaAI_Operator.html#id2).
+
+### Verify via web console
+1.	Verify CR by checking the status of **Workloads** -> **DaemonSet** -> **habana-ai-module-device-plugin-xxxxx**.
+2.	Now `DeviceConfig` is created.
+
+### Create CR via CLI
+Apply the CR yaml file:
+```
+oc apply -f  https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/gaudi/gaudi_device_config.yaml
+```
+
+### Verify the DeviceConfig CR is created
+Verify the Gaudi resources are up and running using below command:
```

Comment:
You can use the command below to verify that the DeviceConfig CR has been created.